test/templates/html/spec_runner.html.erb

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Templates Test</title>
+    <link rel="stylesheet" href="/node_modules/mocha/mocha.css" />
+  </head>
+  <body>
+    <!-- setup for mocha -->
+    <div id="mocha"></div>
+    <script src="/node_modules/mocha/mocha.js"></script>
+    <script src="/node_modules/chai/chai.js"></script>
+    <script>
+      mocha.setup('bdd');
+      mocha.reporter('html');
+    </script>
+
+    <!-- load templates -->
+    <script src="/templates/console.js"></script>
+
+    <!-- find and load test cases -->
+    <% glob("spec/**/*_spec.js") do |spec| %>
```

Comment:
If you had the `PROJECT_ROOT` constant being a path_name, you could have probably dropped the whole helper here and used it instead.